### DISCUSSION TITLE
[Feature] keep selection active

### DISCUSSION THREAD

**Discussion by xfaustus** [immich-app/immich#3551] [Feature Request]
### The feature

Usually i select photos to add in proper Album and then i archive them to hide from gallery.
It would be great if after i add photos to the album the selections is not reset so i can just click on archive and avoid to reselect them again.
Thanks

### Platform

- [ ] Server
- [X] Web
- [ ] Mobile

**Comment by lordkitsuna** [comment]
this would indeed be great, better yet for the given usecase would even be a little checkbox for "archive after inserted to album" or something to that effect

**Comment by patienttruth** [comment]
It seems as of 2025-01-14 that the original inquiry (albums) is fixed.

I did notice that assets are deselected after applying a tag: ( My Duplicate discussion https://github.com/immich-app/immich/discussions/15341#discussion-7824115)

Currently when I make a large selection and apply a tag my selection is deselected. This breaks a flow which might be to Archive the photos, or assign them to an album. This is dissimilar to the flow assigning photos to an album where the photos remain selected. (The original example in this discussion).

**Comment by bo0tzz** [comment]
#2686
